scenario/ArbitrumBridgeScenario.ts

```diff
@@ -0,0 +1,213 @@
+    );
+    const upgradeL2GovContractsProposal = utils.defaultAbiCoder.encode(
+      ['address[]', 'uint256[]', 'string[]', 'bytes[]'],
+      [
+        [configurator.address, proxyAdmin.address, proxyAdmin.address],
+        [0, 0, 0],
+        [
+          'setGovernor(address,address)',
+          'deployAndUpgradeTo(address,address)',
+          'transferOwnership(address)'
+        ],
+        [setGovernorCalldata, deployAndUpgradeToCalldata, transferOwnershipCalldata]
+      ]
+    );
+
+    expect(await proxyAdmin.owner()).to.eq(oldLocalTimelock.address);
+    expect(await comet.governor()).to.eq(oldLocalTimelock.address);
+
+    await fastL1ToArbitrumGovernanceExecute(
+      upgradeL2GovContractsProposal,
+      oldBridgeReceiver as ArbitrumBridgeReceiver,
+      world
+    );
+
+    expect(await proxyAdmin.owner()).to.eq(newLocalTimelock.address);
+    expect(await comet.governor()).to.eq(newLocalTimelock.address);
+
+    // Update aliases now that the new Timelock and BridgeReceiver are official
+    await dm.putAlias('timelock', newLocalTimelock);
+    await dm.putAlias('bridgeReceiver', newBridgeReceiver);
+
+    // Now, test that the new L2 governance contracts are working properly via another cross-chain proposal
+    const currentTimelockDelay = await newLocalTimelock.delay();
+    const newTimelockDelay = currentTimelockDelay.mul(2);
+
+    const setDelayCalldata = utils.defaultAbiCoder.encode(['uint'], [newTimelockDelay]);
+    const pauseCalldata = await calldata(comet.populateTransaction.pause(true, true, true, true, true));
+    const l2ProposalData = utils.defaultAbiCoder.encode(
+      ['address[]', 'uint256[]', 'string[]', 'bytes[]'],
+      [
+        [newLocalTimelock.address, comet.address],
+        [0, 0],
+        ['setDelay(uint256)', 'pause(bool,bool,bool,bool,bool)'],
+        [setDelayCalldata, pauseCalldata]
+      ]
+    );
+
+    expect(await newLocalTimelock.delay()).to.eq(currentTimelockDelay);
+    expect(currentTimelockDelay).to.not.eq(newTimelockDelay);
+
+    await fastL1ToArbitrumGovernanceExecute(l2ProposalData, newBridgeReceiver, world);
+
+    expect(await newLocalTimelock.delay()).to.eq(newTimelockDelay);
+    expect(await comet.isAbsorbPaused()).to.eq(true);
+    expect(await comet.isBuyPaused()).to.eq(true);
+    expect(await comet.isSupplyPaused()).to.eq(true);
+    expect(await comet.isTransferPaused()).to.eq(true);
+    expect(await comet.isWithdrawPaused()).to.eq(true);
+  }
+);
+
+async function fastL1ToArbitrumGovernanceExecute(
+  l2ProposalData: string,
+  arbitrumBridgeReceiver: ArbitrumBridgeReceiver,
+  world: World
+) {
+  const governanceDeploymentManager = world.auxiliaryDeploymentManager;
+  if (!governanceDeploymentManager) {
+    throw new Error('cannot execute governance without governance deployment manager');
+  }
+
+  const isMainnetArbitrum = world.base.network === 'arbitrum';
+  const compWhale = isMainnetArbitrum ? COMP_WHALES.mainnet[0] : COMP_WHALES.testnet[0];
```

Comment:
rebase after https://github.com/compound-finance/comet/pull/720 is merged; rework to avoid this logic in the scenario itself

Reply:
rebased, refactored to incorporate #722